server/app/services/ti/DateOfBirthNotInPastException.java

```diff
@@ -0,0 +1,7 @@
+package services.ti;
+
+public class DateOfBirthNotInPastException extends Exception {
```

Comment:
If we keep the exception classes rather than returning `CiviFormError` from the service method, mark this and the other classes as `final` so that the decision to allow subclassing is explicit rather than implicit.